src/subclass/widget.rs

```diff
@@ -1490,6 +1490,12 @@ unsafe extern "C" fn widget_size_allocate<T: ObjectSubclass>(
     imp.size_allocate(&wrap, allocate);
 }
 
+/// Realize the given widget
+///
+/// # Safety
+///
+/// If the widget is not yet in a toplevel window then according to GTK,
+/// "bad things will happen".
 pub unsafe extern "C" fn widget_realize<T: ObjectSubclass>(ptr: *mut gtk_sys::GtkWidget)
```

Comment:
All these functions shouldn't be `pub` at all. Thanks for noticing :)

Reply:
Aah, should I remove the safety notes and unpub them?

Reply:
Yes please